src/tasks/daily.ts

```diff
@@ -365,6 +368,22 @@ export function configureSnojo(): void {
   }
 }
 
+function determineFreeEmbezzlerBanish(): void {
+  if (freeBanishesChecked) return;
+  const useFreeBanishes =
+    getBanishedMonsters().get($item`ice house`) !== $monster`fluffy bunny` &&
+    gregLikeFightCount() < 120 && // 60 turns of banish from mafia middle finger ring, and 30 x 2 from two snokebombs
+    gregLikeFightCount() > 0 &&
+    have($item`mafia middle finger ring`) &&
+    !get("_mafiaMiddleFingerRingUsed") &&
+    have($skill`Snokebomb`) &&
+    get(`_snokebombUsed`) === 0
+      ? true
```

Comment:
It feels kind of silly to use ternary to decide a boolean; why would you do `condition ? true : false` when you can just do `condition`?